qtplaskin/database.py

```diff
@@ -45,11 +45,18 @@ def get_molecule(state):
     
     return state.split('(')[0].split('^')[0]
 
-def get_molar_mass(state):
+def get_molar_mass(state, warn_err=True):
     
     molecule = get_molecule(state)
     atoms_dict = get_atoms_in(molecule)
-    return sum([_molar_mass_dict[k]*v for k,v in atoms_dict.items()])
+    try:
+        return sum([_molar_mass_dict[k]*v for k,v in atoms_dict.items()])
```

Comment:
To fix the Warning on electrons issue (see my comment in the PR) you could : 

1. Add : 

`for k,v in atoms_dict.items() if k != 'E'`

2. Or define molar mass of the electrons in _molar_mass_dict

_molar_mass_dict['E'] = 0.  # electron mass is neglected

I like 2. slightly better.